.github/workflows/changelog.yml

```diff
@@ -0,0 +1,42 @@
+name: changelog
+
+on:
+  pull_request:
+    paths:
+      - 'CHANGELOG.md'
+      - 'xtask/**/*'
+
+env:
+  #
+  # Dependency versioning
+  #
+
+  # This is the MSRV used by `wgpu` itself and all surrounding infrastructure.
+  REPO_MSRV: "1.88"
+
+  #
+  # Environment variables
+  #
+
+  CARGO_INCREMENTAL: false
+  CARGO_TERM_COLOR: always
+  RUST_LOG: info
+  RUST_BACKTRACE: '1'
+  CACHE_SUFFIX: c # cache busting
+
+jobs:
+  changelog:
+    timeout-minutes: 1
```

Comment:
def should be like 2-3 minutes - this is worst case behavior and there's always some random slowdowns so don't want this to totally fail wildly.

Reply:
I've just seen wild things happen with CI, not really a strong opinion, just vibes are better with 2